src/components/favorites-tab.yml

```diff
@@ -11,5 +11,14 @@ targetDeletedMessage:
   en: The target of this link has been deleted
   de: Das Ziel dieses Links wurde gelöscht
 info:
-  en: Here you can find all documents, rooms and users that you have marked as favorite.
-  de: Hier finden Sie alle Dokumente, Räume und Benutzer, die Sie als Favorit markiert haben.
+  en: Here you can find all users, rooms and documents that you have marked as favorite.
+  de: Hier finden Sie alle Benutzer, Räume und Dokumente, die Sie als Favorit markiert haben.
+favoriteUsers:
+  en: Favorite users
+  de: Favorite Benutzer
```

Comment:
"Benutzer, die Sie als Favorit hinzugefügt haben" (probably than also align English "Users you have added to your favorites". Also I realized the english "add **to** favorites" would be also better in German "**zu** den Favoriten hinzufügen" (the "als" is OK, though).

Reply:
I see 🤔 The "als" phrasing matches currently the tooltips for the favorite star "Als Favorit hinzufügen". If you think "zu den Favoriten hinzufügen" sounds better, I will change in both places.

Meantime, updated the headlines. Thanks!

Reply:
Cool, thanks!